packages/cli/src/commands/update.js

```diff
@@ -0,0 +1,316 @@
+import fs from 'fs'
+import path from 'path'
+
+import boxen from 'boxen'
+import latestVersion from 'latest-version'
+import { Listr } from 'listr2'
+import semver from 'semver'
+import terminalLink from 'terminal-link'
+
+import { errorTelemetry } from '@redwoodjs/telemetry'
+
+import { getPaths } from '../lib'
+import c from '../lib/colors'
+
+import { validateTag } from './upgrade'
+
+export const command = 'update'
```

Comment:
I'm not sure this should be it's own command. If anything, it could be part of `yarn rw check`, which runs diagnostics on a project. (It could do a lot more—one of those things could be the version.) But don't make any major architectural changes yet! Just a thought

Reply:
Does `check` have any effect - i.e. does it fix any issues from the diagnostics?

My main aim was to have a command which shouldn't actually touch the users code. Instead only fetching data in the background and triggering behind the scenes effects. That was so we don't have a command running in the background potentially going mad and introducing silent, very hard to track down issues into the users project.

Reply:
`check` only prints its results to the console. It doesn't modify any files